packages/orchestrator/lib/client.ts

```diff
@@ -111,4 +139,28 @@ export class OrchestratorClient {
             }
         }
     }
+
+    public async executeAction(props: TExecuteActionProps): Promise<TExecuteReturn> {
+        const { args, ...rest } = props;
+        const schedulingProps = {
+            ...rest,
+            args: {
+                ...args,
+                type: 'action' as const
+            }
+        };
+        return this.execute(schedulingProps);
+    }
+
+    public async executeWebhook(props: TExecuteWebhookProps): Promise<TExecuteReturn> {
+        const { args, ...rest } = props;
+        const schedulingProps = {
+            ...rest,
+            args: {
+                ...args,
+                type: 'webhook' as const
+            }
+        };
+        return this.execute(schedulingProps);
+    }
```

Comment:
I went with a function for each type (an `executeSync` will be added soon) instead of a single function with a Union type parameters since types are already quite complex